### DISCUSSION TITLE
Map legend

### DISCUSSION THREAD

**Discussion by danieldegroot2** [xspanger3770/GlobalQuake#80] [Q&A]
Add a map legend which explains the colours the application is using, if possible. Could alternatively be added in the readme.

There seems to be a frequency graph available in the application (below), but this is something else entirely.
![image](https://github.com/xspanger3770/GlobalQuake/assets/67521919/bf7e45f4-8314-4b77-b14c-4cca4cd7abf1)

**Comment by xspanger3770** [comment]
The feature you are referring to is a magnitude histogram for a given earthquake. Each station gives one magnitude reading and the final magnitude is calculated as median value from all of them.

**Comment by danieldegroot2** [comment]
Ok. Would it be possible to add a list to the readme to show the colours used for the dots/list shown elsewhere on the map and -what I assume is- the corresponding final magnitude? just the integer values, so 0,1,2,3,... . This is just for documentation.

**Comment by xspanger3770** [comment]
Documentation for all the displayed features will be eventually added, but right now I want to concentrate more on the actual coding than writing documentation...
